android/src/main/java/io/sentry/RNSentryModule.java

```diff
@@ -190,4 +219,114 @@ private Level logLevel(int level) {
                 return Level.OFF;
         }
     }
+
+    @ReactMethod
+    public void setUser(final ReadableMap user) {
+        if (user == null) {
+            Sentry.setUser(null);
```

Comment:
Can we use `configureScope` instead? Ideally we'd drop these top level methods eventually.

```suggestion
            Sentry.configureScope(s -> { s.setUser(null) });
```

Same for other `Sentry.setUser` calls below